src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticAnalyzerAPIUsageAnalyzer.cs

```diff
@@ -11,6 +11,8 @@
 using System.Diagnostics;
 using Microsoft.CodeAnalysis.Operations;
 using Analyzer.Utilities.PooledObjects;
+using static System.Linq.ImmutableArrayExtensions;
```

Comment:
All these using additions should be reverted.

Reply:
Reverted!